js/scenes/Home/HomeContainer.js

```diff
@@ -16,7 +16,7 @@ class HomeContainer extends Component {
       title: 'HOME',
     }
```

Comment:
If not using internal `state`  or  life cycle methods, use functional components. keep it lean :)

Reply:
Unfortunately, our ExNavigator library requires standalone scenes to be classes so that we can attach the static route options to them.